### PR TITLE
[Extra] Using latest components under extra in composer.json

### PR DESCRIPTION
**Pull request by samsonasik**
Signed-off-by: Abdul Malik Ikhsan <samsonasik@gmail.com>

|    Q          |   A
|-------------- | ------
| QA            | yes

### CODE REVIEW

**Comment by samsonasik** [comment]
The extra components that not installable under PHP 8 are:

1. laminas-mvc-console  < -- deprecated, remove it ?
2. laminas-cache <-- possibly need to be excluded as it may need to wait for version 3.0 ? /cc @boesing 
3. laminas-test  <-- on https://github.com/laminas/laminas-test/pull/18

**Comment by froschdesign** [comment]
@samsonasik 
> 1. laminas-mvc-console  < -- deprecated, remove it ?

The component is deprecated, so the skeleton application should no longer suggest it. 

> 3. laminas-test  <-- on [laminas/laminas-test#18](https://github.com/laminas/laminas-test/pull/18)

It would make sense to create a release for laminas-test with PHP 8 support first. Pull requests for all required changes should be available.

**Comment by samsonasik** [comment]
@froschdesign thank you, I removed laminas-mvc-console from the extra list.

**Comment by Xerkus** [comment]
> The component is deprecated, so the skeleton application should no longer suggest it.

As an aside, do we consider `laminas-cli` package a replacement?

**Comment by froschdesign** [comment]
> …do we consider `laminas-cli` package a replacement?

There is a stable release, so this is a good idea! 👍

**Comment by samsonasik** [comment]
@Xerkus @froschdesign I updated to use laminas-cli https://github.com/laminas/laminas-mvc-skeleton/pull/39/commits/38873926473012c18bc1dfb6387ebf4b58d877dd

File: composer.json [hunk 1]
```diff
@@ -30,78 +30,78 @@
...
+                "constraint": "^1.1.0",
                 "prompt": "Would you like to install the official MVC plugins, including PRG support, identity, and flash messages?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-psr7bridge",
-                "constraint": "^1.2.0",
+                "constraint": "^1.3.1",
                 "prompt": "Would you like to use the PSR-7 middleware dispatcher?"
             },
```
Comment (Xerkus) [inline composer.json:88]: Oh, totally forgot. This should be replaced with `laminas/laminas-mvc-middleware` at `^2.0`

Should I do that in another PR?

Comment (samsonasik) [reply composer.json:88]: @Xerkus yes, that's ok 👍

**Review by Xerkus — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -30,78 +30,78 @@
...
                 "module": true
             },
             {
-                "name": "laminas/laminas-mvc-console",
-                "constraint": "^1.2.0",
-                "prompt": "Would you like to install MVC-based console support? (We recommend migrating to zf-console, symfony/console, or Aura.CLI)",
+                "name": "laminas/laminas-cli",
+                "constraint": "^1.1.1",
+                "prompt": "Would you like to install command-line interface support?",
                 "module": true
```
Comment (Xerkus) [inline composer.json]: Checking `laminas-cli`. It does not provide or need module
```suggestion
```

**Review by Xerkus — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (samsonasik) [reply composer.json]: removed https://github.com/laminas/laminas-mvc-skeleton/pull/39/commits/1dca5cf6ed067f0758c2c4455f51c8c5c8794eac

**Review by Xerkus — APPROVED** [review]
